### PR TITLE
Fix #26: Add global "Hide Images" toggle in Settings

### LINKED ISSUE #26: Add "Block Images" global toggle to Settings
Hide images is a current feature, we should move it and clump it into the general shields settings.

### PR DESCRIPTION
**Pull request by kylehickinson**
Simply integrates with the Firefox "Hide Images" feature for the time being

## Pull Request Checklist

- [ ] My patch has gone through review and I have addressed review comments
- [x] My patch has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!`

- [ ] I have updated the *Unit Tests* to cover new or changed functionality
- [ ] I have updated the *UI Tests* to cover new or changed functionality
- [ ] I have marked the bug with `[needsuplift]`
- [x] I have made sure that localizable strings use `NSLocalizableString()`

## Screenshots

_None included_

## Notes for testing this patch

_None included_

### CODE REVIEW

File: Client/Frontend/Settings/SettingsViewController.swift [hunk 1]
```diff
@@ -271,6 +271,7 @@ class SettingsViewController: TableViewController {
                 BoolRow(title: Strings.Block_Phishing_and_Malware, option: Preferences.Shields.blockPhishingAndMalware),
                 BoolRow(title: Strings.Block_Scripts, option: Preferences.Shields.blockScripts),
                 BoolRow(title: Strings.Fingerprinting_Protection, option: Preferences.Shields.fingerprintingProtection),
+                BoolRow(title: Strings.AppMenuNoImageMode, option: Preferences.Shields.blockImages),
```
Comment (kylehickinson) [inline Client/Frontend/Settings/SettingsViewController.swift:274]: Will need a copy update if we choose to change it from "Hide Images" to "Block Images"

**Review by tmancey — APPROVED** [review]
👍

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 2]
```diff
@@ -889,7 +889,7 @@ class BrowserViewController: UIViewController {
             return
         }
         if #available(iOS 11, *) {
```
Comment (jhreis) [inline Client/Frontend/Browser/BrowserViewController.swift:891]: Delete this.

**Review by jhreis — COMMENTED** [review]
This wired in? Small PR, but seem to be missing where the pref is actually changed? 😬

**Comment by kylehickinson** [comment]
It's wired in yeah, as of now it uses the FF image blocking implementation until we implement #159 (seen here: https://github.com/brave/brave-ios/pull/158/files#diff-221fd685cced62e8c7406c0b5f24ed6cR1070)

File: Client/Frontend/Browser/BrowserViewController.swift (hunk 2 above)
Comment (kylehickinson) [reply Client/Frontend/Browser/BrowserViewController.swift:891]: This will be done with #165

**Pull request merged**